### PR TITLE
[PUB-1680] Composer sidebar wrapper component

### PR DESCRIPTION
**Pull request by MayaUribe**
## Description

Creating new component for a sidebar that will be placed next to the composer, with the intention to use it for hashtag management.

## Context & Notes

Currently our users are mostly using the first comment section to add Hashtag, which is why we are creating this new component to hopefully facilitate this process for them 👌

## Screenshots
Image 2019-07-18 at 11 23 40 AM

## Checklist

-   [x] My code follows the code style and guidelines of this project. 
-   [x] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have tested different user stories. 
-   [x] I have identified similar or related features and tested they are still working.
-   [ ] I have performed a self-review of my own code.
-   [x] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [x] I have commented my code, particularly in hard-to-understand areas.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

File: packages/composer-sidepanel/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { connect } from 'react-redux';
+import ComposerSidepanel from './components/ComposerSidepanel';
+
+export default connect(
+  state => ({
+    userData: state.appSidebar.user,
```
Comment (karelm) [inline packages/composer-sidepanel/index.js]: I don't think we need the userData here, so feel free to remove it

**Comment by karelm** [comment]
@MayaUribe I think theres a padding issue with the icon in the top right corner, but otherwise its looking good 🙌  
image

**Comment by MayaUribe** [comment]
> @MayaUribe I think theres a padding issue with the icon in the top right corner, but otherwise its looking good 🙌
> image

Oh good catch @karelm!

**Comment by anafilipadealmeida** [comment]
Seems good to me! Just one question: since this is basically just a ui component file, would it make more sense to add to the share-components, instead of having a dedicated package with a high-order-component in the index.js?

**Comment by MayaUribe** [comment]
> Seems good to me! Just one question: since this is basically just a ui component file, would it make more sense to add to the share-components, instead of having a dedicated package with a high-order-component in the index.js?

Sounds fair enough to me @anafilipadealmeida! it is just a container after all, I'll move it to the share-components folder 😄

**Pull request merged**